modules/transfer-frontend/ecs.tf

```diff
@@ -157,7 +157,8 @@ data "aws_iam_policy_document" "frontend_ecs_execution" {
       "arn:aws:ecr:eu-west-2:${data.aws_ssm_parameter.mgmt_account_number.value}:repository/transfer-frontend",
       "${aws_cloudwatch_log_group.aws-otel-collector.arn}:*",
       "arn:aws:ecr:eu-west-2:${data.aws_ssm_parameter.mgmt_account_number.value}:repository/aws-otel-collector",
-      "arn:aws:logs:eu-west-2:${data.aws_caller_identity.current.account_id}:log-group:/aws/ecs/application/metrics:log-stream:otel-stream-*"
+      "arn:aws:logs:eu-west-2:${data.aws_caller_identity.current.account_id}:log-group:/aws/ecs/application/metrics:log-stream:otel-stream-*",
+      "arn:aws:logs:eu-west-2:${data.aws_ssm_parameter.mgmt_account_number.value}:log-group:/aws/events/ecs-task-events-${var.environment}:*"
```

Comment:
The account number should be the environment account number as each log group is held at the account level. Therefore should reference: `data.aws_caller_identity.current.account_id`